### PR TITLE
fixed gradle, please check last commits

### PR DESCRIPTION
**Pull request by ConnorHeidema**
So unfortunately I merged commits right into master -_- but fortunately there was nothing bad but unfortunately I could push right to master but the protection on the branch means I can't undo it...

What all my past commits were were basically adding a port constant, and turning the colours that may be sent into an enum to simplify some of the logic. I also updated the gradle although the URL still points at the same place because the public facing URL isn't collecting the plugin for some reason.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -7,31 +7,38 @@ buildscript {
...
-    }
-    repositories {
-        maven {
-            url maven_repo
-            credentials {
-                username artifactory_app_username
-                password artifactory_app_password
+            artifactory_app_username = "${rightmesh_artifactory_username}"
+            artifactory_app_password = "${rightmesh_artifactory_password}"
+            artifactory_app_key = "${rightmesh_hellomesh_key}"
```
Comment (keeferrourke) [inline app/build.gradle]: `${rightmesh_ripple_key}`, not `hellomesh`.

File: app/build.gradle [hunk 2]
```diff
@@ -7,31 +7,38 @@ buildscript {
...
+
+            rightmesh_maven_repo = "https://research.rightmesh.io/artifactory/libs-local"
+            maven_repo = "${rightmesh_maven_repo}"
```
Comment (keeferrourke) [inline app/build.gradle]: why the two variables here? you could directly assign `maven_repo` to artifactory URI.

**Review by keeferrourke — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 2 above)
Comment (ConnorHeidema) [reply app/build.gradle]: Fixed to only have one variable

File: app/build.gradle (hunk 1 above)
Comment (ConnorHeidema) [reply app/build.gradle]: done

**Review by keeferrourke — APPROVED** [review]

**Pull request merged**